openapi-cli/src/test/java/io/ballerina/openapi/generators/client/AllOfResponsesTests.java

```diff
@@ -55,7 +55,7 @@ public void getReturnTypeForAllOf() throws IOException, BallerinaOpenApiExceptio
         TypeHandler.createInstance(response, false);
         Operation post = response.getPaths().get("/users/{userId}/meetings").getPost();
         FunctionReturnTypeGeneratorImp functionReturnType = new FunctionReturnTypeGeneratorImp(post, response, "post");
-        assertEquals(functionReturnType.getReturnType().get().type().toString(), "inline_response_201|error");
+        assertEquals(functionReturnType.getReturnType().get().type().toString(), "inline_response_201|error?");
```

Comment:
What is the response schema here?

Reply:
This is correct, we have three responses: 201 with body, 300 and 400. The `nil` is created for the 300 response